### PR TITLE
onlykey: init at 5.3.3

### PR DESCRIPTION
**Pull request by kalbasit**
###### Motivation for this change

Add [OnlyKey](https://github.com/trustcrypto/OnlyKey-App) at v5.3.3.

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [21.11 Release Notes (or backporting 21.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2111-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/tools/security/onlykey/generate.sh [hunk 1]
```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env nix-shell
+#! nix-shell -i bash -p nodePackages.node2nix
+
+# XXX: --development is given here because we need access to gulp in order to build OnlyKey.
+exec node2nix --nodejs-10 --development -i pkg.json -c onlykey.nix -e ../../../development/node-packages/node-env.nix --no-copy-node-env
```
Comment (SuperSandro2000) [inline pkgs/tools/security/onlykey/generate.sh:5]: Can we use a newer node version?
```suggestion
exec node2nix --nodejs-10 --development -i package.json -c onlykey.nix -e ../../../development/node-packages/node-env.nix --no-copy-node-env
```

File: pkgs/tools/security/onlykey/default.nix [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+  script = writeShellScript "${onlykey.packageName}-starter-${onlykey.version}" ''
+    ${node_webkit}/bin/nw ${onlykey}/lib/node_modules/${onlykey.packageName}/build
+  '';
+
+  desktop = makeDesktopItem {
+    name = "${onlykey.packageName}";
+    exec = script;
+    icon = "${onlykey}/lib/node_modules/${onlykey.packageName}/resources/onlykey_logo_128.png";
+    desktopName = "${onlykey.packageName}";
+    genericName = "${onlykey.packageName}";
```
Comment (SuperSandro2000) [inline pkgs/tools/security/onlykey/default.nix:47]:
```suggestion
    name = onlykey.packageName;
    exec = script;
    icon = "${onlykey}/lib/node_modules/${onlykey.packageName}/resources/onlykey_logo_128.png";
    desktopName = onlykey.packageName;
    genericName = onlykey.packageName;
```

File: pkgs/tools/security/onlykey/generate.sh (hunk 1 above)
Comment (kalbasit) [reply pkgs/tools/security/onlykey/generate.sh]: Yep, thanks for the reminder. I moved to using the latest supported by node2nix, `--nodejs-14`.

**Review by SuperSandro2000 — APPROVED** [review]
should be good to go

**Pull request merged**